bin/localnet/deployer/deployer.go

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/MysteriumNetwork/payments/cli/helpers"
+	"github.com/MysteriumNetwork/payments/mysttoken/generated"
+	generated2 "github.com/MysteriumNetwork/payments/promises/generated"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"math/big"
+	"os"
+	"time"
+)
+
+func main() {
+	flag.Parse()
```

Comment:
There are flags You are parsing?

Reply:
Scattered in helpers which are imported from payments lib :( it was not a good idea to register params as "globals". Will improve payments lib and come back here

Reply:
Extracted as list of args before flag parse.